src/Views/NightLightView.vala

```diff
@@ -0,0 +1,178 @@
+/*
+* Copyright (c) 2017 elementary LLC (https://elementary.io)
+*
+* This program is free software; you can redistribute it and/or
+* modify it under the terms of the GNU General Public
+* License as published by the Free Software Foundation; either
+* version 2 of the License, or (at your option) any later version.
+*
+* This program is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+* General Public License for more details.
+*
+* You should have received a copy of the GNU General Public
+* License along with this program; if not, write to the
+* Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+* Boston, MA 02110-1301 USA
+*/
+
+public class Display.NightLightView : Gtk.Grid {
+    private Gtk.Scale temp_scale;
+
+    public int temperature {
+        set {
+            temp_scale.set_value (value);
+        }
+    }
+
+    construct {
+        var settings = new GLib.Settings ("org.gnome.settings-daemon.plugins.color");
+
+        var status_label = new Gtk.Label (_("Night Light:"));
+        status_label.halign = Gtk.Align.END;
+        status_label.xalign = 1;
+
+        var status_switch = new Gtk.Switch ();
+        status_switch.halign = Gtk.Align.START;
+        status_switch.hexpand = true;
+
+        var description_label = new Gtk.Label (_(
+            "Night Light makes the colors of your display warmer. This may help prevent eye strain and sleeplessness."
```

Comment:
I feel like putting the underscore function in the same line than the string would be more elegant